test/source/tests/settings.ts

```diff
@@ -245,6 +245,12 @@ export let defineSettingsTests = (testVariant: TestVariant, testWithBrowser: Tes
       await SettingsPageRecipe.addKeyTest(t, browser, 'ci.tests.gmail@flowcrypt.dev', unprotectedPrvKey, 'this is a new passphrase to protect previously unprotected key');
     }));
 
+    ava.default('settings - error modal when page parameter invalid', testWithBrowser('ci.tests.gmail', async (t, browser) => {
+      const invalidParamModalPage = await browser.newPage(t, TestUrls.extension(`chrome/settings/index.htm?page=invalid`));
```

Comment:
Maybe it's missing acctEmail param in the url?

Reply:
Added. That made it not close right away! I don't know why - when I test manually it works just fine even if I leave acctEmail blank. Oh well, that did the trick! Additionally, HTML tags don't render in modals (which is good) so I had to find the test a different way than using the data-test tag. Now tests are passing!